spinta/commands/write.py

```diff
@@ -212,15 +221,26 @@ def _stream_group_key(data: DataItem):
     return data.model, data.prop, data.backend, data.action
 
 
-def is_streaming_request(request: Request):
+def get_content_type_from_request(request: Request):
     content_type = request.headers.get('content-type')
     if content_type:
         content_type = cgi.parse_header(content_type)[0]
+    return content_type
+
+
+def is_streaming_request(content_type):
+    content_type = cgi.parse_header(content_type)[0]
     return content_type in STREAMING_CONTENT_TYPES
 
 
+def is_not_streaming_request(content_type):
+    content_type = cgi.parse_header(content_type)[0]
+    return content_type not in STREAMING_CONTENT_TYPES
```

Comment:
Maybe use `detect_backend_from_content_type`? If backend could not be detected, then proceed to other request processing options?